mp/src/game/client/momentum/client_events.cpp

```diff
@@ -34,6 +41,13 @@ bool CMOMClientEvents::Init()
         }
     }
 
+    UnloadConVarOrCommand("retry");
+
+    static ConCommand retry("retry", []()
+    {
+        engine->ExecuteClientCmd("reload");
+    });
```

Comment:
Any particular reason this wasn't done with the others in server_events?

Reply:
`retry` is a client command and `reload` doesn't work when executed on the server.